private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -1,19 +1,24 @@
 import React from "react";
+import { connect } from "react-redux";
 
 import { RootState } from "../../store/rootReducer";
-import { connect } from "react-redux";
+import { UserType } from "../../store/auth/types";
+import { BatchTemplatesState } from '../../store/templates/types';
 import { setPage } from "../../store/page/actions";
+import { getAllTemplates } from "../../store/templates/actions";
 
 import requireAuthentication from "../../utils/requireAuthentication";
 import Gallery from "../Gallery";
-import { Title, DashboardContainer } from "../Dashboard/styled";
 import PreviewModal from "./PreviewModal";
 import SearchPage from './SearchPage/SearchPage';
 
+import { Title, DashboardContainer } from "../Dashboard/styled";
+
 const mapStateToProps = (state: RootState) => {
   return {
     isAuthenticated: state.auth.isAuthenticated,
     user: state.auth.user,
+    batchTemplates: state.batchTemplates,
```

Comment:
nit: would rename to 'allTemplates'.

Reply:
I had to pull Kodys changes in order to do something in my PR, These are his changes. I will merge with Dev since his change was just approved